src/comlink.ts

```diff
@@ -496,7 +501,15 @@ function createProxy<T>(
         }).then(fromWireValue);
         return r.then.bind(r);
       }
-      return createProxy(ep, pendingListeners, [...path, prop]);
+
+      const cachedProxy = cachedProxies.get(prop);
```

Comment:
Is `path` not important to consider here? Why is it provided to `createProxy` but not something we need for the lookup key?

Reply:
`path` is only relevant when creating the proxy, but not when accessing a property through the `get` proxy method.